nodeEditor/src/diagram/graphCanvas.tsx

```diff
@@ -858,7 +857,15 @@ export class GraphCanvasComponent extends React.Component<IGraphCanvasComponentP
             for (var frameData of editorData.frames) {
                 var frame = GraphFrame.Parse(frameData, this, editorData.map);
                 this._frames.push(frame);
-            }
+            }  
+        }
+    }
+
+    addFrame(editorData: IEditorData) {
+        if (editorData.frames) {
+            const frame = GraphFrame.Parse(editorData.frames[editorData.frames.length - 1], this, editorData.map);

```

Comment:
why not editorData.frames[0] ?

Reply:
because if the user imports multiple frames in a row, the frame that needs to be added to the editor is at the end of editorData.frames.